src/sentry/utils/batching_kafka_consumer.py

```diff
@@ -326,6 +326,7 @@ def _flush(self, force=False):
         )
 
         batch_results_length = len(self.__batch_results)
+        self.__record_timing("batch.size", batch_results_length)
```

Comment:
Is there a reason to record this as a timing? It looks more like a gauge or incr.

Reply:
Mostly to get aggregations ("avg", "max", etc.) automatically.
Term "timing" might be a bit misleading here, but for historical reasons, we already use it across the board to capture numerical data, because under the hood it's based on historgram/distribution.